src/styles/theme.scss

```
@@ -39,25 +40,36 @@ $spacingLarge: 24px;
-$drawerWidth: 350px;
-$drawerPullerWidth: 90px;
 $chartCardMaxWidth: 500px;
 $iconButtonWidthHeight: 22px;
 
+/** Layout primitives (C235 redesign) **/
+$sidebarWidth: 48px;
+$panelWidth: 315px;
+$panelLeftDefault: 56px;
+$panelBottomGap: 8px;
+$panelMobileGutter: 8px;
+// Derived top offsets — anchored to $headerHeight so C236 only changes that token.
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)